hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/contract/ITestAbfsFileSystemContractSeek.java

```diff
@@ -68,6 +68,9 @@ protected Configuration createConfiguration() {
   protected AbstractFSContract createContract(final Configuration conf) {
     conf.setInt(AZURE_READ_AHEAD_RANGE, MIN_BUFFER_SIZE);
     conf.setInt(AZURE_READ_BUFFER_SIZE, MIN_BUFFER_SIZE);
+    // Disabling cache to make sure new configs are picked up.
```

Comment:
pull out into an azure test utils class, just as `S3ATestUtils.disableFilesystemCaching()does

Reply:
Great suggestions. Taken

Reply:
Already had a AbfsTestUtils class defined. Added there itself.
I noticed that class has other methods without any usage. Not sure how but was wondering should they be removed or not.